### PR TITLE
docs: add docs for connection quality

### PR DESCRIPTION
**Pull request by raviteja83**

### CODE REVIEW

File: docs/javascript/v2/advanced-features/peer-connection-quality.mdx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: Peer Connection Quality
+nav: 5.06
+---
+
+You can get a score of current user's connection quality when in preview, post join connection quality will be available for all peers.
+The score ranges from 1-5, 5 being the hightest(good network health) and 1 being the lowest(bad network health).
+-1 indicates the sdk is not able to determine the network health within reasonable time.
```
Comment (raviteja83) [inline docs/javascript/v2/advanced-features/peer-connection-quality.mdx:8]: should we use -1 or 0. currently we are returning -1 in the published sdk.

Comment (triptu) [reply docs/javascript/v2/advanced-features/peer-connection-quality.mdx:8]: -1, as that is what server sends for undefined. the lowest from server is 0 which means disconnected(no traffic on ice)

File: docs/javascript/v2/advanced-features/peer-connection-quality.mdx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    }
+}, selectConnectionQualityByPeerID(peerId));
+```
+
+</Tab>
+
+<Tab id='connection-quality-jsx'>
+
+```jsx
+const downlinkScore = useHMSStore(selectConnectionQualityByPeerID(peerId))?.downlinkScore;
```
Comment (triptu) [inline docs/javascript/v2/advanced-features/peer-connection-quality.mdx:39]: the field is named downlinkQuality now

File: docs/javascript/v2/advanced-features/peer-connection-quality.mdx [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+title: Peer Connection Quality
+nav: 5.06
+---
+
+You can get a score of current user's connection quality when in preview, post join connection quality will be available for all peers.
+The score ranges from 1-5, 5 being the hightest(good network health) and 1 being the lowest(bad network health).
+-1 indicates the sdk is not able to determine the network health within reasonable time.
+
```
Comment (triptu) [inline docs/javascript/v2/advanced-features/peer-connection-quality.mdx:9]: another section for "Measure Network Quality in Preview" can be added

Comment (triptu) [reply docs/javascript/v2/advanced-features/peer-connection-quality.mdx:9]: with a brief description about how it works from - https://www.100ms.live/docs/android/v2/features/Network-Quality-Reports

**Review by triptu — APPROVED** [review]

**Pull request merged**